nitro-protocol.tex

```diff
@@ -180,7 +180,7 @@ \subsection{Offloading Virtual Channels}
   \begin{align}
     \adj{\holds{L'}{x}{}, \holds{G}{x}{\guar{J}{\rchi, A, I}}, \; \holds{J}{}{\alloc{\rchi: x, I: x}}}
   \end{align}
-  \item Finally $A$ calls $\claim{G}{A}{\rchi}$ to move the funds from $G$ to $\rchi$.
+  \item Finally $A$ calls $\claim{G}{\rchi}{x}$ to move the funds from $G$ to $\rchi$.
```

Comment:
The original syntax is wrong:


![Screenshot 2021-04-20 at 18 18 23](https://user-images.githubusercontent.com/1833419/115438021-d01c0c00-a204-11eb-9415-09608b2ffdad.png)

should be 

![Screenshot 2021-04-20 at 18 20 10](https://user-images.githubusercontent.com/1833419/115438244-11acb700-a205-11eb-826d-8c7dd9e5f72a.png)